Reverse buddy display name to "name (address)"

crackcomm opened the pull request:
I think it's better.
![Proposed display name](http://i.imgur.com/xG1G0.png)

crackcomm committed:
Reverse buddy display name to "name (address)"

crackcomm committed:
Reverse buddy display name to "name (address)"

prof7bit commented:
Idea is ok. Maybe It should be a config option, I remember having experimented with some other possible display formats in some very early versions many years ago but for some reason then I found it better (can't remember why exactly) to have it this way. Maybe it shoud be configurable, I guess many people are used to it now as it and would want to keep it. I'm a bit busy with the android port at the moment (and also other things), sorry for the long delay without answer.

crackcomm committed:
Reverse buddy display name to "name (address)"

Option gui::buddies_display_format when 0, on buddies list display format is old, when 1 display is in new format "name (address)".

crackcomm committed:
Reverse buddy display name to "name (address)"

crackcomm commented:
It can be now modified by "gui"::"buddies_display_format" option. 0 is old format, 1 is new format as shown on screenshot :)
Im waiting for your android version ;)
